flang/include/flang/Frontend/CompilerInvocation.h

```diff
@@ -22,6 +21,7 @@
 #include "flang/Parser/parsing.h"
 #include "flang/Semantics/semantics.h"
 #include "mlir/Support/Timing.h"
+#include "flang/Support/LangOptions.h"
```

Comment:
The switched order here makes the linter [unhappy](https://github.com/llvm/llvm-project/pull/124416#issuecomment-2615853379).
```suggestion
#include "flang/Support/LangOptions.h"
#include "mlir/Support/Timing.h"
```